lib/phoenix/live_dashboard/live/home_live.ex

```diff
@@ -38,11 +38,14 @@ defmodule Phoenix.LiveDashboard.HomeLive do
       system_usage: system_usage
     } = SystemInfo.fetch_system_info(socket.assigns.menu.node)
 
+    environment = SystemInfo.fetch_environment_info(socket.assigns.menu.node, session["env_keys"])
```

Comment:
Oh, good call. Let's pass the session["env_keys"] to the existing `fetch_system_info`, so we have less back and forth across the nodes. :)